packages/mobile/src/identity/verification.ts

```diff
@@ -512,18 +512,43 @@ function* tryRevealPhoneNumber(
     )
     if (!response.ok) {
       const body = yield response.json()
-      Logger.error(TAG + '@tryRevealPhoneNumber', `Reveal response not okay: ${body.error}`)
-      ValoraAnalytics.track(AnalyticsEvents.verification_reveal_error, {
-        issuer,
-        statusCode: response.status,
-      })
-      throw new Error(
-        `Error revealing to issuer ${attestation.attestationServiceURL}. Status code: ${response.status}`
-      )
+
+      if (body.error.includes('No incomplete attestation found')) {
+        // Retry as attestation service might not yet have received the block where it was made responsible for an attestation
+        yield call(
```

Comment:
I would add a sleep here as it might take a second or two for the service to catch up (since we suspect it to be behind)

Reply:
+1, a `yield delay(3000)` here makes sense I think